src/components/controls/Controls.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Controls.css';
+
+const counter = (name, count) => {
```

Comment:
you shouldn't need to do this here - this file should stay pretty much the same as it was in the starter code.